@xen-orchestra/lite/src/stories/web-core/state-hero/vts-state-hero.story.vue

```diff
@@ -2,9 +2,9 @@
   <ComponentStory
     v-slot="{ properties, settings }"
     :params="[
-      prop('type').required().enum('page', 'card', 'panel').preset('card').widget(),
+      prop('type').required().enum('page', 'card', 'panel', 'table').preset('card').widget(),
       prop('busy').bool().widget(),
-      prop('image').enum('no-result', 'under-construction', 'no-selection', 'error').widget(),
+      prop('image').enum('no-result', 'under-construction', 'no-selection', 'error-no-data').widget(),
```

Comment:
Why did you rename `error` to `error-no-data`? (in DS, the image name is `error`
If the renaming is intentional, don't forget to delete the `error.svg` file